README.md

```diff
@@ -1,23 +1,25 @@
-# InfluxData Helm Charts
+# InfluxData Helm charts
```

Comment:
Shouldn't titles be title case?

Reply:
We use sentence case for headings. Typically, we follow [Google's dev style guide]( https://developers.google.com/style).